src/lib/decorators.ts

```diff
@@ -55,9 +60,9 @@ export const customElement = (tagName: keyof HTMLElementTagNameMap) =>
  * corresponding attribute value. A `PropertyDeclaration` may optionally be
  * supplied to configure property features.
  */
-export const property = (options?: PropertyDeclaration) => (proto: Object,
-                                                            name: string) => {
-  (proto.constructor as typeof UpdatingElement).createProperty(name, options);
+export const property = (options?: PropertyDeclaration): FixedPropertyDecorator => (proto: Object,
```

Comment:
to satisfy typescript (temporarily) and be consistent with how other decorators are written, this should probably return `any`.